Doc/library/sqlite3.rst

```diff
@@ -25,6 +25,15 @@ The sqlite3 module was written by Gerhard Häring.  It provides an SQL interface
 compliant with the DB-API 2.0 specification described by :pep:`249`, and
 requires SQLite 3.7.15 or newer.
 
+This document includes four main sections:
+
+* :ref:`sqlite3-tutorial`: teaches how to use the sqlite3 module
```

Comment:
```suggestion
* :ref:`sqlite3-tutorial` teaches how to use the sqlite3 module
```

IMO, it reads better and sounds a little friendlier if these are phrased as prose sentences